docs/spec-files/out_of_band.md

```diff
@@ -20,7 +20,7 @@ The `id` of the message passed in a URL or a QR code is used to as the `pthid` o
 
 Each message passed this way must be contained within an `out-of-band` message, as described below.
 
-The out-of-band protocol a single message that is sent by the *sender*.
+The out-of-band protocol consists in a single message that is sent by the *sender*.
```

Comment:
`consists of` instead of `consists in`?

Reply:
It seemed to be really an intrinsic property to me. Since you merged already I guess this is fine?